src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -135,6 +148,53 @@
                   </button>
                 </span>
               </form>
+              <form
+                #editForumUrl="ngForm"
+                class="form-inline"
+                fxLayout
+                *ngIf="!(isNFL$ | async) && workflow?.mode !== WorkflowType.ModeEnum.DOCKSTOREYML"
+              >
+                <div class="form-group" fxFlex="noshrink" fxLayout fxLayoutAlign=" center">
+                  <strong matTooltip="Link to a discussion forum for this workflow">Discussion Forum</strong>:
+                  <a *ngIf="!forumUrlEditing" [href]="workflow?.forumUrl" fxFlexOffset="4px"> {{ workflow?.forumUrl }} </a>
+                  <input
+                    *ngIf="forumUrlEditing"
+                    minlength="3"
+                    maxlength="256"
+                    type="text"
```

Comment:
The type should ideally be `url`. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

But we don't use this anywhere else, so it's worth checking how it behaves. If it works, I don't think you need minLength. If you do keep minLength, it should be at least 7 (http://).